jsonpatcher/jsonpatcher.go

```diff
@@ -47,7 +60,6 @@ func init() {
 	cbornode.RegisterCborType(patchEvent{})
 	cbornode.RegisterCborType(recordEvents{})
 	cbornode.RegisterCborType(operation{})
-	cbornode.RegisterCborType(time.Time{})
```

Comment:
Turns out `time.Time` was not properly being cbor encoded, so I switched the handling to unix nanos.